### DISCUSSION TITLE
Some keys are not working

### DISCUSSION THREAD

**Discussion by ikspekt** [seerge/g-helper#1640] [Q&A] [answered]
Sorry I solved problem by reinstalling driver
I don't know how to erase this post
Thank you very much

**Comment by seerge** [comment, marked as answer]
@ikspekt does previous final release (0.129) work for you ?

And post a log from 0.134 here

**Reply by ikspekt** [reply]
Wow u r so fast,, I solved it
Always thank you for great program
